tools/Ensembl-REST/get_gene/get_gene_info.xml

```diff
@@ -0,0 +1,56 @@
+<tool id="get_gene_info_wrapper" name="get_gene_info_from_Ensembl" version="0.1.1">
```

Comment:
`name` is free text, I'd use `"Get gene info from Ensembl"`. And just `using REST API` in `<description>`.